tracer/src/Datadog.Trace.Tools.Analyzers/Datadog.Trace.Tools.Analyzers.csproj

```diff
@@ -10,6 +10,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.11.0" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="5.0.0" PrivateAssets="all" />
+    <!-- BannedApiAnalyzers needs to flow to consuming projects, so don't mark as PrivateAssets -->
+    <PackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.3.4" />
```

Comment:
_Should_ this be in the analyzers project? 🤔 I think we should reference it in the Directory.Build.props file instead?